tasks/write-to-a-windows-event-log/src/main.rs

```diff
@@ -0,0 +1,44 @@
+#[cfg(target_os = "windows")]
+mod bindings {
+    ::windows::include_bindings!();
+}
+
+const TOKEN_READ: u32 = 0x00020008;
+const EVENTLOG_WARNING_TYPE: u16 = 2;
+
+#[cfg(target_os = "windows")]
+fn main() {
+    use bindings::windows::win32::security::*;
+    use bindings::windows::win32::system_services::*;
+
+    unsafe {
```

Comment:
I prefer to limit the scope of `unsafe` to the FFI calls, like

```
unsafe { OpenProcessToken(...) };
```

Reply:
Good. That's seems like a good practice, thanks.